framework/tlvf/yaml/tlvf/ieee_1905_1/tlvDeviceInformation.yaml

```diff
@@ -21,5 +21,8 @@ sInfo:
   _type: struct
   mac: sMacAddr
   media_type: eMediaType
+# MMZ Media-specific information
+# 1 octet: Number of octets in ensuing field
+# n octets: Media-specific information of the local interface (as defined in the "Media-specific information" column of Table 6-12.
```

Comment:
Many of the IEEE1905.1 structures are currently unused. Some of them will be revived and corrected as part of #300.